docs/node-operators/lifecycle-payment.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Lifecycle of a Payment
+hide_title: true
+---
+
+:::note
+
+A new version of Mina Docs is coming soon! This page will be rewritten.
+
+:::
+
+# The Lifecycle of a Payment
+
+In Mina, payments pass through several steps before they are considered verified and complete. This document is meant to walk through what happens to a single payment in a simplified overview to help users understand a little about how Mina payments work. It it not a comprehensive technical overview, but instead a simplified walkthrough for users. For a more detailed technical overview aimed at developers wishing to understand the codebase check out the [technical lifecycle of a payment](./architecture/lifecycle-payment-technical/this-link-didn't-exist-in-old-docs).
```

Comment:
@jasongitmail This link didn't exist in old docs. Should we get rid of the last sentence entirely?

Reply:
Isn't that this page? https://docs.minaprotocol.com/en/node-operators/lifecycle-payment